core/common/src/main/java/alluxio/underfs/UnderFileSystem.java

```diff
@@ -15,8 +15,12 @@
 import alluxio.Configuration;
 import alluxio.Constants;
 import alluxio.PropertyKey;
+import alluxio.exception.ExceptionMessage;
```

Comment:
Yes. I think @gpang preferred this way. I don't have a strong opinion on this. I can revert to the way it was if you guys prefer.